packages/itwin/grouping-mapping-widget/src/components/Groups/Editing/GroupAction.tsx

```diff
@@ -174,15 +158,43 @@ export const GroupAction = (props: GroupActionProps) => {
       setCurrentStep(GroupActionStep.QueryBuilder);
       setShowValidationMessage(false);
       props.onSaveSuccess();
-    } catch (error: any) {
-      handleError(error.status);
-    } finally {
-      setIsLoading(false);
-    }
-  }, [validator, setShowValidationMessage, query, simpleSelectionQuery, getAccessToken, props, mappingClient, iModelId, details, clearPresentationSelection]);
+      await queryClient.invalidateQueries({ queryKey: ["groups"] });
+    },
+  });
 
   const isQueryBuilderStep = currentStep === GroupActionStep.QueryBuilder;
   const isGroupDetailsStep = currentStep === GroupActionStep.GroupDetails;
+
+  const isLoading = isSaveLoading || isQueryLoading;
+
+  const isUpdating = isSaveLoading || isRendering;
+
+  const isBlockingActions = !(
```

Comment:
Couldn't this be just `isRendering || isLoading`?

Reply:
De Morgan's Laws applied.